repos/spack_repo/builtin/packages/libmetatensor_torch/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+
+from spack.package import *
+
+
+class LibmetatensorTorch(CMakePackage):
+    """TorchScript/C++ bindings to metatensor"""
+
+    homepage = "https://docs.metatensor.org"
+    url = "https://github.com/metatensor/metatensor/releases/download/metatensor-torch-v0.7.6/metatensor-torch-cxx-0.7.6.tar.gz"
+    git = "https://github.com/metatensor/metatensor.git"
+
+    maintainers("HaoZeke", "luthaf", "rmeli")
+    license("BSD-3-Clause", checked_by="HaoZeke")
+
+    version("0.7.6", sha256="8dcc07c86094034facba09ebcc6b52f41847c2413737c8f9c88ae0a2990f8d41")
+    version("0.7.5", sha256="24611998e076dd0a782a9792f26b315cc57514013715e8a0aedf2a3a891ba1b4")
+    version("0.7.4", sha256="356c412d60a88dfe995976eaad3378ea0e0a858bbca8d43ed299b44c7403fefb")
+    version("0.7.3", sha256="ea46e44ba3c7379e1134960851fc6cfd0b0f9de1e7154f60c36f4dfbe59449e3")
```

Comment:
Is there significant value in adding these checksums to the new package as well?

If need be, in special cases, users can always use `spack checksum -a` to add more checksums, while in general, the spack concretizer does not have to consider those versions:
```suggestion
```